flang/test/Fir/vector-always.fir

```diff
@@ -0,0 +1,42 @@
+// RUN: %flang_fc1 -emit-llvm -o - %s | FileCheck %s
```

Comment:
I sort of wanted to test that this will pass all the way through to LLVM IR. I'm not sure if that is tested on the `cf` dialect. Maybe it would be better to add the lowering here from fir to cf and then add a test in cf to LLVM IR for the loop annotation?

Reply:
The `--fir-to-llvm-ir` conversion includes cf to LLVM Dialect. That should be a sufficient test for this. The LLVM Dialect to LLVM IR for loop attributes is already (hopefully) tested elsewhere.
An alternative is to write a test in the Integration directory.